Sources/xcbeautify/Xcbeautify.swift

```diff
@@ -0,0 +1,29 @@
+import ArgumentParser
+import Foundation
+import XcbeautifyLib
+
+
+struct Xcbeautify: ParsableCommand {
+	@Flag(name: [.short, .long], help: "Only print tasks that have warnings or errors.")
+	var quiet = false
+	
+	@Flag(name: [.customLong("qq", withSingleDash: true), .long], help: "Only print tasks that have errors.")
```

Comment:
Shouldn't this be `.customShort("qq)`? And the long version would still be `--quiter`.

Reply:
customShort only allows a single character. I switched the order, so usage looks like this: `USAGE: xcbeautify [--quiet] [--quieter] [--is-ci]` instead of suggesting `[-qq]` on the second place.